pkg/oc/cli/cmd/rollout/cancel.go

```diff
@@ -112,6 +112,7 @@ func (o CancelOptions) Run() error {
 		config, ok := info.Object.(*deployapi.DeploymentConfig)
 		if !ok {
 			allErrs = append(allErrs, kcmdutil.AddSourceToErr("cancelling", info.Source, fmt.Errorf("expected deployment configuration, got %T", info.Object)))
```

Comment:
can you use kapi.Scheme.ObjectKind(info.Object).Kind ? seems nicer than raw go type